__tests__/supertest.js

```diff
@@ -87,7 +101,7 @@ describe('Wallet integration tests', () => {
         .send({
           tokens: [seed.token.uuid],
           sender_wallet: seed.wallet.name,
-          receiver_wallet: seed.walletB.name,
+          receiver_wallet: seed.walletB.id,
```

Comment:
Because integration is heavy, we need to construct a lot of stuff to build a case for testing, so I think it's better to test different detailed cases in UNIT test, and use integration(supertest here) to test the major workflow. So here we can test every different input in the transferRouter.spec.js, it would be also easier to build case for this in a unit test by using mock I think.

Reply:
Got you, integration tests should only test the use cases, and this code increases the cost to test too much. 
Would you prefer me to move the auth test too into the .spec too, or is that ok where it is?

Reply:
For the time being, I am fine with the auth test there, but with time the integration test getting bigger, we need to consider a good way to organize all these tests, to make it more clear I think :)